physics/PBL/SATMEDMF/satmedmfvdifq.F

```diff
@@ -2376,8 +2379,14 @@ subroutine satmedmfvdifq_run(im,km,ntrac,ntcw,ntrw,               &
          enddo
       enddo
       do i = 1,im
-        dusfc(i) = -1.*rho_a(i)*stress(i)*u1(i,1)/spd1(i)
-        dvsfc(i) = -1.*rho_a(i)*stress(i)*v1(i,1)/spd1(i)
+        if(icplocn2atm == 0) then
```

Comment:
If icplocn2atm is used a switch, can you use a logical (true/false) instead of an integer (0/1)?

Reply:
Please see the response above. Thanks!